systems/robotInterfaces/@Biped/planSwingPitched.m

```diff
@@ -196,23 +150,13 @@ function add_foot_origin_knot(swing_pose, speed)
 % Apex knot 1
 max_terrain_ht = max(terrain_pts_in_toe_local(3,:));
 toe_ht_in_local = max_terrain_ht + params.step_height;
-toe_pos_in_local = interp1([0, 1], [mean(swing1_toe_points_in_toe_local, 2), mean(swing2_toe_points_in_toe_local, 2)]', APEX_FRACTIONS(1))';
-constraints = {posture_constraint,...
-               WorldQuatConstraint(biped, swing_body_index, quat_toe_off, quat_tol),...
-               WorldPositionInFrameConstraint(biped,swing_body_index,...
-                    mean(swing_toe_points_in_foot, 2), T_toe_local_to_world, [toe_pos_in_local(1); -LATERAL_TOL; toe_ht_in_local], [toe_pos_in_local(1); LATERAL_TOL; toe_ht_in_local])};
-[pose, q_latest] = solve_for_pose(constraints, q_latest);
+pose = [swing1_origin_pose(1:3)*(1-APEX_FRACTIONS(1)) + APEX_FRACTIONS(1)*swing2_origin_pose(1:3) + [0;0;toe_ht_in_local]; quat2rpy(quat_toe_off)];
```

Comment:
I think this isn't exactly right. Previously, we were interpolating the initial and final toe positions using the `APEX_FRACTIONS` to find the desired position of the toe, then solving for the pose of the origin. This is instead interpolating between the initial and final origin positions using those same fractions. That's certainly equivalent if the foot doesn't rotate, but the foot *does* rotate, so there may be some discrepancy. Specifically, if the origin of the foot link is far from the toe, this interpolation may result in the foot moving too far forward or even moving backward.